pkg/gateways/store/postgres/transfers/update_by_id.go

```diff
@@ -0,0 +1,27 @@
+package transfers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/daniel1sender/Desafio-API/pkg/domain/entities"
+)
+
+const UpdateByIDStatement = `INSERT INTO transfers(
```

Comment:
this is not intended to be used outside of this package:
```suggestion
const updateByIDStatement = `INSERT INTO transfers(
```